config/boards/licheepi-4a.wip

```diff
@@ -0,0 +1,17 @@
+# RISC-V LicheePi 4A
+BOARD_NAME="LicheePi 4A"
+BOARDFAMILY="thead"
+BOARD_MAINTAINER="chainsx"
```

Comment:
Property `BOARD_MAINTAINER` is auto-pulled from db. Put some data here https://www.armbian.com/maintainer-registry/ so we have additional way to contact and to list you [here](https://www.armbian.com/authors/). This, nickname, GitHub user and which board you maintained is minimum. Recommended ideas: https://docs.armbian.com/Board_Maintainers_Procedures_and_Guidelines/